antsibull/utils/get_pkg_data.py

```diff
@@ -0,0 +1,14 @@
+# Author: Felix Fontein <felix@fontein.de>
+# Author: Toshio Kuratomi <tkuratom@redhat.com>
+# License: GPLv3+
+# Copyright: Ansible Project, 2020
+"""General functions for working with aiohttp."""
+
+import pkgutil
+
+
+def get_pkg_data(filename: str, package: str = 'antsibull.data') -> bytes:
```

Comment:
Since this is just a light wrapper around `pkgutil.get_pkg_data`, we should probably maintain the order of parameters (package, filename).

Reply:
The main disadvantage of using the original order (and why I wasn't doing that) is that the whole idea of having the default value `antsibull.data` for package won't work anymore. If you really want to change that, I can do it though.

Reply:
This is a good point and I was very torn on what it should do.  I think I've come to a conclusion, though.

We can either think of this function as being generic or being specific to antsibull.

If it's generic, then `(package, filename)` with no default makes sense because both pieces of information are required to uniquely identify the piece of data.  Since package is the container and filename is the exact key inside of the container listing them in that order is us zeroing in on data by first determining the neighborhood it's in and then its exact address within there.  This is probably why `pkgutil.get_data()` has its parameters in this order.

On the other hand, if it's specific to antsibull, then there's not a reason to even have package in the parameter list.  We can assume the package will always be "the data directory inside of antsibull, aka: `antsibull.data`".  However, this starts to break down if we have other data directories inside of antsibull... The same things I said about a generic API would start to apply again and the parameters should probably have an offset from `antsibull` as the first argument followed by the filename inside of the offset and we're back where we started.  (We already have another data directory, `antsibull.data.docsite`.  It's just that jinja2 takes care of loading data from there rather than the loading happening in our code).

So I think following the generic function model makes the most sense:
```suggestion
def get_pkg_data(package: str, filename: str) -> bytes:
```